common/v2/components/TransactionFlow/ConfirmTransaction.tsx

```diff
@@ -111,7 +111,7 @@ export default function ConfirmTransaction({
   /* Get contact info */
   const recipientContact = getContactByAddressAndNetwork(receiverAddress, network);
   const senderContact = getContactByAccount(account);
-  const senderAccount = getAccount(account);
+  const senderAccount = getAccount(account) || account;
```

Comment:
If the ConfirmPanel needs to support sending from an account that is not included in the users accounts, we need to:
- document the use case (commenting in the code?)
- rename the variables to reflect the case (it's no longer an 'account' and is just an address since we know neither the network,  balance nor assets associated with this)
- Update the types. Strong typing should have prevented this.
- `txConfig` was intended to share information between steps with the assumption that we would only interact with  existing accounts. We need to adapt this.

Reply:
Removed `senderAccount `and introduced `sender`. The sender contains all required props to go through the confirm/broadcast flow and some optional ones to display more info (account balance, adding tx to account, etc...).